pkg/api/subscription.go

```diff
@@ -22,11 +22,32 @@ const (
 	SubscriptionStateWarned       SubscriptionState = "Warned"
 	SubscriptionStateSuspended    SubscriptionState = "Suspended"
 	SubscriptionStateDeleted      SubscriptionState = "Deleted"
+
+	// SubscriptionFeatureE2E is the feature in the subscription that is used
+	// to indicate if we need to save ARO cluster config into the E2E StorageAccount
+	SubscriptionFeatureE2E string = "Microsoft.ContainerService/SaveAROTestConfig"
 )
 
 // SubscriptionProperties represents subscription properties
 type SubscriptionProperties struct {
 	MissingFields
 
-	TenantID string `json:"tenantId,omitempty"`
+	TenantID           string                        `json:"tenantId,omitempty"`
+	AccountOwner       AccountOwnerProperties        `json:"accountOwner,omitempty"`
+	RegisteredFeatures []RegisteredFeatureProperties `json:"registeredFeatures,omitempty"`
+}
+
+// AccountOwnerProperties represents the subscription account owner informations
+type AccountOwnerProperties struct {
+	MissingFields
+
+	Email string `json:"email,omitempty"`
```

Comment:
Added this because you mentioned you may want to encrypt this later, but im not using it part of this PR.